keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -1039,8 +1115,21 @@ For each of them, fill in the following information by copying the below templat
     - Testing: Are there any tests for failure mode? If not, describe why.
 -->
 
+None.
+
 ###### What steps should be taken if SLOs are not being met to determine the problem?
 
+- Make sure that the node is running with CRI runtime which supports this feature.
+- Make sure that `crictl info` (with the latest crictl)
+  reports that `supplemental_groups_policy` is supported.
```

Comment:
It needs to update `crictl` to display `features` field in `crictl info` command.